include/sapi/tss2_sys.h

```diff
@@ -55,17 +55,14 @@ typedef struct _TSS2_SYS_OPAQUE_CONTEXT_BLOB TSS2_SYS_CONTEXT;
 // Input structure for authorization area(s).
 //
 typedef struct {
-    uint8_t cmdAuthsCount;
-    TPMS_AUTH_COMMAND **cmdAuths;
-} TSS2_SYS_CMD_AUTHS;
+    uint16_t          count;
```

Comment:
i'm noticing changes using stdint,h directly rather than UINIT16, are the defined sizes in the tss going to be replaced by stdint directly at some point?

Im just thinking consitency here, why define UINT16 if it's not going to be used in leiu of uint16_t?

Reply:
This is a long standing debate and I think I asked the same question when I started participating in the TSS working group. The answer I got boils down to "This is the way the TPM spec does things and we follow the TPM spec as closely as possible". IMHO it's an unnecessary level of indirection which is what I think you're getting at.

Reply:
Sorry I misunderstood your point. I think I'm with you now: all of the other structures from the spec use UINT* so why are we using the stdint.h types here. I think @AndreasFuchsSIT was just using the closest type to the one we already had in the structure.

Reply:
Well, I took it from the TSS Spec https://trustedcomputinggroup.org/wp-content/uploads/TSS_SAPI_v1.1_r21_Public_Review.pdf page 42.

@flihp @williamcroberts I don't know, why we made it such, but for the moment it is, I suppose.

Shall I change it nevertheless ?

Reply:
@williamcroberts : you asked the question originally as part of your review. Is the justification given sufficient or are you requesting changes?

Reply:
@williamcroberts this is still waiting for your input